contributors/devel/pull-requests.md

```diff
@@ -108,29 +108,29 @@ Here's the process the PR goes through on its way from submission to merging:
 
 If you're **not** a member of the Kubernetes organization:
 
-1. Reviewer/Kubernetes Member checks that the PR is safe to test. If so, they comment `@k8s-bot ok to test`
+1. Reviewer/Kubernetes Member checks that the PR is safe to test. If so, they comment `/ok-to-test`
 1. Reviewer suggests edits
 1. Push edits to your PR branch
 1. Repeat the prior two steps as needed
 1. (Optional) Some reviewers prefer that you squash commits at this step 
 1. Owner is assigned and will add the `/approve` label to the PR
 
-If you are a member, or a member comments `@k8s-bot ok to test`, the pre-submit tests will run:
+If you are a member, or a member comments `/ok-to-test`, the pre-submit tests will run:
 
 1. Automatic tests run. See the current list of tests on the [MERGE REQUIREMENTS tab, at this link](https://submit-queue.k8s.io/#/info)
 1. If tests fail, resolve issues by pushing edits to your PR branch
-1. If the failure is a flake, a member can comment `@k8s-bot [e2e|unit] test this issue: #<flake issue>`
+1. If the failure is a flake, anyone on trusted PRs can comment `/retest` to rerun failed tests
 
 Once the tests pass, all failures are commented as flakes, or the reviewer adds the labels `lgtm` and `approved`, the PR enters the final merge queue. The merge queue is needed to make sure no incompatible changes have been introduced by other PRs since the tests were last run on your PR.
 
 Either the [on call contributor](on-call-rotations.md) will manage the merge queue manually, or the [GitHub "munger"](https://github.com/kubernetes/test-infra/tree/master/mungegithub) submit-queue plugin will manage the merge queue automatically.
 
 1. The PR enters the merge queue ([http://submit-queue.k8s.io](http://submit-queue.k8s.io))
-1. The merge queue triggers a test re-run with the comment `@k8s-bot test this`
-    1. Author has signed the CLA (`cla: yes` label added to PR)
+1. The merge queue triggers a test re-run with the comment `/test all [submit-queue is verifying that this PR is safe to merge]`
+    1. Author has signed the CLA (`cncf-cla: yes` label added to PR)
     1. No changes made since last `lgtm` label applied
 1. If tests fail, resolve issues by pushing edits to your PR branch
-1. If the failure is a flake, a member can comment `@k8s-bot [e2e|unit] test this issue: #<flake issue>`
+1. If the failure is a flake, anyone can comment `/retest`
```

Comment:
This isn't true if the PR hasn't been marked okay (e.g. a single test run was started with `/test all` or by `/lgtm`, without the PR itself being marked trusted).